apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -430,7 +435,27 @@ class BehandlingFactory(
         } else if (sak.sakType == SakType.BARNEPENSJON) {
             "${persongalleri.soesken.size} søsken"
         } else if (sak.sakType == SakType.OMSTILLINGSSTOENAD) {
-            val barnUnder20 = persongalleri.soesken.count { Folkeregisteridentifikator.of(it).getAge() < 20 }
+            val foedselsdatoForSoesken =
+                runBlocking {
+                    coroutineScope {
+                        persongalleri.soesken
+                            .map {
+                                async {
+                                    pdlTjenester
+                                        .hentPdlModellForSaktype(
+                                            it,
+                                            PersonRolle.BARN,
```

Comment:
```suggestion
                                            PersonRolle.TILKNYTTET_BARN,
```
siden de er søsken så burde vi hente det mer minifiserte objektet

Reply:
Ja egentlig, men hvor viktig er den merknaden for SB?

Reply:
Har vi en god løsning å bare hente en opplysningstype per person fra PDL da?